lib/class-wp-rest-block-types-controller.php

```diff
@@ -430,12 +430,12 @@ public function get_item_schema() {
 					'readonly'    => true,
 				),
 				'parent'        => array(
-					'description' => __( 'Parent blocks, defaults to empty it no parents', 'gutenberg' ),
+					'description' => __( 'Parent blocks.', 'gutenberg' ),
 					'type'        => 'array',
```

Comment:
We would also have change this to support the a null value. Some like this `array( 'null', 'object' )`

Reply:
Is 01de805 correct?